agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -1343,11 +1347,19 @@ func makeClient(t *testing.T, srv pbpeerstream.PeerStreamServiceServer, peerID s
 	require.NoError(t, err)
 	receivedSub2, err := client.Recv()
 	require.NoError(t, err)
+	receivedSub3, err := client.Recv()
+	require.NoError(t, err)
+
+	// This is required when the client subscribes to server address replication messages.
+	// We assert for the handler to be called at least once but the data doesn't matter.
+	srv.mockSnapshotHandler.expect("", 0, 0, nil)
 
-	// Issue a services and roots subscription pair to server
+	// Issue services, roots, and server address subscription to server.
+	// Note that server address may not come as an initial message
```

Comment:
This message is misleading; if we mock data in the `expect` above we should see an initial server addresses

Reply:
todo